js/src/modal.js

```diff
@@ -471,6 +471,10 @@ class Modal {
       // Adjust fixed content padding
       SelectorEngine.find(SELECTOR_FIXED_CONTENT)
         .forEach(element => {
+          if (window.innerWidth > element.clientWidth + this._scrollbarWidth) {
```

Comment:
Maybe we should move this to a private function?

Reply:
Can you make the update, please? @muhamadamin1992

Reply:
Yes) Sorry I wrote comment above 3 weeks ago, but forgot submit(

function setModalSpaces({ element, attributeName, cssPropName }) {
  if (window.innerWidth > element.clientWidth + this._scrollbarWidth) {
    return
  }

  const actualPadding = element.style[attributeName]
  const calculatedPadding = window.getComputedStyle(element)[cssPropName]
  Manipulator.setDataAttribute(element, cssPropName, actualPadding)
  element.style[attributeName] = `${Number.parseFloat(calculatedPadding) + this._scrollbarWidth}px`
}